bootstrap/src/main/java/io/lumigo/javaagent/LumigoAgent.java

```diff
@@ -33,8 +35,14 @@ public static void agentmain(final String agentArgs, final Instrumentation inst)
       System.setProperty("lumigo.debug", "true");
     }
     if (is_switch_off()) {
+      System.err.println(
+          "Lumigo OpenTelemetry JavaAgent distribution disabled via the 'LUMIGO_SWITCH_OFF' environment variable");
```

Comment:
Message is inconsistent with what we print out in other distros: https://github.com/lumigo-io/opentelemetry-js-distro/blob/11f5dfc0af9372ff1455e10adf0ed709932b9acf/src/wrapper.ts#L64

Reply:
Then you have inconsistency in python as well. this is what you specified in the issue
https://github.com/lumigo-io/opentelemetry-python-distro/blob/943847ffac919f7fe3a60adce90b3640a151786e/src/lumigo_opentelemetry/__init__.py#L74